app/models/job.rb

```diff
@@ -3,4 +3,7 @@ class Job < ActiveRecord::Base
   STATUSES = [:draft, :published, :archived]
   enumerize :state, in: STATUSES
   belongs_to :organization
+  belongs_to :published_by, foreign_key: :user_id, class_name: "User"
```

Comment:
Here is a rookie question :)   Why not do it like this `belongs_to :user`?
I'm sure there is a good reason for doing it like this, I just wanna be educated, that's all.